docs/release-notes.rst

```diff
@@ -5,6 +5,205 @@ This document describes major installer-related changes in Fedora releases.
 
 A guide on adding new entries is in the release documentation.
 
+Fedora 40
+#########
+
+Changes in the graphical interface
+----------------------------------
+
+Dir and image installations run only in the non-interactive text mode now
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Anaconda now requires a fully defined kickstart file for installations into a local image
+(via the ``--image`` cmdline option) or a local directory (via the ``--dirinstall`` cmdline
+option) and these installations can run only in a non-interactive text-based user interface.
+The ``anaconda`` and ``livemedia-creator`` tools can be used for these types of installations
+with the following changes:
+
+    * If a user requests a dir or image installation, Anaconda runs in the text mode.
+    * If the user doesn't specify a kickstart file, Anaconda reports an error and aborts.
+    * If the specified kickstart file is incomplete, Anaconda reports an error and aborts.
+    * All options for specifying the user interface are ignored.
+
+See also:
+    - https://fedoraproject.org/wiki/Changes/Anaconda_dir_and_image_installations_in_automated_text_mode
+    - https://github.com/rhinstaller/anaconda/pull/5447
+
+Remove support for additional repositories from GUI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The widget that allowed users to specify and edit additional repositories for the package
+installation was removed from the "Installation Source" screen of the GTK-based graphical
+user interface. Use the kickstart support or the ``inst.addrepo`` boot option to specify
+additional repositories.
+
+See also:
+    - https://github.com/rhinstaller/anaconda/pull/5448
+
+Redesign the Time & Date spoke in GUI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The timezone map was removed from the Time & Date spoke in the GTK-based user interface
+and the spoke was redesigned to accommodate the changes. The installer no longer depends
+on the the ``libtimezonemap`` package.
+
+See also:
+    - https://github.com/rhinstaller/anaconda/issues/5404
+    - https://github.com/rhinstaller/anaconda/discussions/5355
+
+Remove support for the LUKS version selection from GUI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All widgets for the LUKS version selection were removed from the "Manual Partitioning"
+screen of the GTK-based graphical user interface. The installer will use the ``luks2``
+version by default for all new devices and keep the LUKS version of existing ones. Use
+the kickstart support or Blivet GUI to select the LUKS version.
+
+See also:
+    - https://github.com/rhinstaller/anaconda/pull/5395
+
+Remove libgnomekbd
+^^^^^^^^^^^^^^^^^^
+
+libgnomekbd is a C library used by Anaconda to display the keyboard
+preview widget. It is stuck in GTK 3 and X11 (libxklavier).
+
+Replace it with Tecla.
```

Comment:
This wording is a bit inconsistent (using imperative) but I don't care to block.

Reply:
Fixed.